### PR TITLE
Test throttle selector UI component

### PR DESCRIPTION
**Pull request by atomgomba**
This is a UI component for selecting throttle value in the motor setup dialogs. Currently it is not functional in the sense that throttle value changes do not take effect because actual motor control is better understood by @limonspb so take this as half-baked strictly UI feature to free limon from the hassle of dealing with HTML/CSS.

Screenshot from 2022-01-20 10-22-10

Screenshot from 2022-01-20 10-21-43

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by limonspb** [comment]
@atomgomba 
plus check a couple of sonar warnings plz here
image

**Comment by Asizon** [comment]
This PR seems a needed fix to make the new motor order dialog working properly since latest changes on how idle works on firmware. So i have asigned to 10.8.0 milestone.

**Review by Asizon — CHANGES_REQUESTED** [review]
Windows enviroment builds are failing here:

```
D:\a\1\s\src\components\shared\TestThrottleSelector\TestThrottleSelectorComponent.js
  4:26  error  Parsing error: Unexpected token =
```

**Comment by limonspb** [comment]
> Windows enviroment builds are failing here:
> 
> ```
> D:\a\1\s\src\components\shared\TestThrottleSelector\TestThrottleSelectorComponent.js
>   4:26  error  Parsing error: Unexpected token =
> ```

yeah i think our Chrome version is not supporting static fields yet.

**Comment by atomgomba** [comment]
It's a pitty but it seems like if the supported EVMA version is different in the test env than the app? 😳

**Comment by limonspb** [comment]
> It's a pitty but it seems like if the supported ECMA version is different in the test env than the app? 😳

When I started working on presets I also wanted to use static class variables. But after some compatibility researsh I found I'd better not to yet. I don't remember why tho :)

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by haslinghuis** [comment]
See: https://chromestatus.com/feature/6482797915013120

We are using:
```
navigator.appVersion.match(/.*Chrome\/([0-9\.]+)/)[1];
"91.0.4472.114"
```

According to NW.js we should be on Node 16.4.0 already:
```
        {
            "version": "v0.54.1",
            "date": "2021/07/02",
            "files": ["win-x64", "win-ia32", "linux-x64", "linux-ia32", "osx-x64"],
            "flavors": ["normal", "sdk"],
            "components": {
                "node": "16.4.0",
                "chromium": "91.0.4472.114"
            }
        },
```

File: src/components/shared/TestThrottleSelector/TestThrottleSelectorComponent.js
Comment (haslinghuis) [inline src/components/shared/TestThrottleSelector/TestThrottleSelectorComponent.js:60]:
```suggestion
                    const btn = $(itemTpl.replace("{{ label }}", `${opt}%`))
```

**Review by limonspb — APPROVED** [review]

**Review by blckmn — DISMISSED** [review]

**Comment by atomgomba** [comment]
I cannot merge for some reason

**Comment by limonspb** [comment]
> I cannot merge for some reason

what do you mean?

**Review by blckmn — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Pull request closed without merging**
